### PR TITLE
Add a bunch of parsers

### PR DESCRIPTION
**Pull request by jfear**
Along with the SRA project I ended up writing a bunch of different parsers for different tools. I have tested them by looking at example output, but have not written any formal tests yet. This is similar to the odo stuff we looked at before, but simpler. Each parser takes a `sample` which is added as an index (for easy concatenation) and a `file`, and returns a data frame. 

This allows the simple idiom of:

```python
rule agg_featurecounts:
    input: targets['featurecounts']
    output: pattern['featurecounts_agg']
    run:
        from lcblib.parser.featurecounts import parse_featureCounts_counts as parser
        dfs = []
        for fn in input:
            name = os.path.basename(os.path.dirname(fn))
            dfs.append(parser(name, fn))
        
        pd.concat(dfs).to_csv(str(output), sep='\t')
```

which probably could be turned into a wrapper.

### CODE REVIEW

File: lcdblib/parse/parser.py [hunk 1]
```diff
@@ -0,0 +1,598 @@
+import os
```
Comment (daler) [inline lcdblib/parse/parser.py]: is everything in one file?

File: lcdblib/parse/picard.py [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+def parse_picardCollect_summary(sample, file):
+    """Parser for picard collectRNAMetrics summary.
+
+    Parameters
+    ----------
+    sample: str
+        Sample name which will be added as row index.
+    file: str
+        Path to the fastqc zip file.
```
Comment (daler) [inline lcdblib/parse/picard.py]: s/fastqc/summary file/

Comment (daler) [inline lcdblib/parse/picard.py]: (actually same for all functions in this module)

File: lcdblib/parse/rseqc.py [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+from collections import OrderedDict
+
+def parse_inferExperiment(sample, file):
+    """Parse rseqc infer expeirment.
+    Parameters
+    ----------
+    sample: str
+        Sample name which will be added as row index.
+    file: str
+        Path to the fastqc zip file.
```
Comment (daler) [inline lcdblib/parse/rseqc.py]: s/fastqc/rseqc etc

**Comment by daler** [comment]
This is awesome! Combined with the recent multiqc customization features, this will give us tons of flexibility with reporting without having to maintain multiqc plugin modules.

**Comment by daler** [comment]
Looks like it's failing because of the new version of pandas which finally removed `Series.sort()`. I'll fix this later today

File: lcdblib/parse/parser.py (hunk 1 above)
Comment (jfear) [inline lcdblib/parse/parser.py]: Oops, did not mean to add this. I was copying things over from that file.

File: lcdblib/parse/picard.py (hunk 2 above)
Comment (jfear) [inline lcdblib/parse/picard.py]: Darn copy and paste. Thought I had caught everything

**Pull request merged**